include/spdlog/sinks/eventlog_sink.h

```diff
@@ -0,0 +1,100 @@
+// Copyright(c) 2015-present, Gabi Melman & spdlog contributors.
+// Distributed under the MIT License (http://opensource.org/licenses/MIT)
+
+// Requires the following registry entries to be present, with the following modifications:

```

Comment:
I saw that the ```add_registry_info``` already creates such entry? why is it "Requires the following registry entries to be present" ?

Reply:
Registering a source is something that requires elevation (aka "running as administrator") so it's usually done during the setup. However I wanted to give the user an option to do it in code.